dali/python/nvidia/dali/pipeline.py

```diff
@@ -135,13 +136,34 @@ class Pipeline(object):
     you will need to call :meth:`start_py_workers` before calling :meth:`build` of any
     of the pipelines. You can find more details and caveats of both methods in Python's
     ``multiprocessing`` module documentation.
+`py_callback_pickler` : module or tuple, default = nvidia.dali.pickling
+    If `py_start_method` is set to *spawn* callback passed to parallel ExternalSource must be picklable.
+    If run in Python3.8 or newer, DALI uses customized pickle (`nvidia.dali.pickling`) when
```

Comment:
Do we want to advertise this module to the users? I think it could be internal.

Reply:
I decided to advertise it because apart from serving as default callback pickler it also contains @pickle_by_value decorator whose whole purpose is to be available for manual usage by users.